config/prow/external-secrets/README.md

```diff
@@ -0,0 +1,17 @@
+## Steps for fetching secret from Secret Manager to k8s cluster
+
+### 1. Deploying external Secrets Operator on cluster
+helm repo add external-secrets https://charts.external-secrets.io
+helm install external-secrets external-secrets/external-secrets -n prow
+
+Note: Though the operator runs in one namespace, crds are available cluster wide and enables us to fetch the secrets in different namespaces.
+Updating the secret store with provider details is what it all needs.
+
+### 2. Creating ibm-secret in cluster for connecting to IBM Cloud Secrets Manager
+`kubectl create secret generic ibm-secret --from-literal=apiKey='API_KEY_VALUE' -n <namespace>`
```

Comment:
this needs to be only in one namespace right?

Reply:
@mkumatag This secret `ibm-secret` is to be present in all namespaces.
The `SecretStore` in all namespaces needs to be updated with ibm provider for us to fetch cert from Secrets Manager.